### DISCUSSION TITLE
Is there support for react-native

### DISCUSSION THREAD

**Discussion by kadirguloglu** [dotnet/aspnetcore#25563] [General]
Can I use the @ microsoft / signalr package in my react-native projects? I found out that there are errors in use and it fails, is an improvement done? If there is no support, I will use socket.io

**Comment by SteveSandersonMS** [comment]
What errors are you encountering? There might be workarounds.

According to its docs, the [SignalR JavaScript client](https://www.npmjs.com/package/@microsoft/signalr) is intended for use in browser and in Node.js. Since it works in those two places, I'd expect it to work in any other relatively standard JavaScript environment, but I'm not aware that React Native specifically has been tested.

cc @BrennanConroy for any other insights here.

**Reply by kadirguloglu** [reply]
I am having a disconnection problem as a result of my tests. Especially it happens after a certain time. The connection is lost for an average of 60-80 seconds.

**Reply by BrennanConroy** [reply]
What errors are you seeing, can you provide network logs?

**Reply by kadirguloglu** [reply]
Since I could not find results for a long time, I continued developing with socket.io
